### PR TITLE
chore(docs): clarify local setup instructions

### PR DESCRIPTION
**Pull request by divyanshu013**
## Description

Clarify a few instructions that I found confusing during local setup for docs site:

- Verify node version. I was unable to build the project on latest node 12
- Specify location of `.env.development` file for docs site

On a related note, even after adding the environment variable in `.env.develpment` file the build process still took time to generate thumbnails. I followed the instructions as mentioned [here](https://github.com/gatsbyjs/gatsby/tree/master/www#running-slow-build-screenshots-placeholder).

Since this looked like a small enough change I considered opening a PR directly. Please let me know if anything else helps. Thanks!

### CODE REVIEW

File: docs/contributing/setting-up-your-local-dev-environment.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Yarn is a package manager for your code, similar to [NPM](https://www.npmjs.com/
 
 - [Install](https://yarnpkg.com/en/docs/install) the Yarn package manager.
 - Ensure you have the latest version of Yarn installed (>= 1.0.2). `yarn --version`
+- Verify your node version, use the LTS version (v10.16.3). `node --version`
```
[the remark this answers is not in the document]
Comment (divyanshu013) [reply docs/contributing/setting-up-your-local-dev-environment.md]: Shall I remove the version number and just say "latest LTS version"?

File: docs/contributing/setting-up-your-local-dev-environment.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ Yarn is a package manager for your code, similar to [NPM](https://www.npmjs.com/
 
 - [Install](https://yarnpkg.com/en/docs/install) the Yarn package manager.
 - Ensure you have the latest version of Yarn installed (>= 1.0.2). `yarn --version`
+- Ensure you have the latest LTS version of Node installed (>= 10.16). `node --version`
```
Comment (pieh) [inline docs/contributing/setting-up-your-local-dev-environment.md]: I didn't realize this PR was open and was addressing https://github.com/gatsbyjs/gatsby/issues/18550 issue (no reference to that issue in this PR), and I went ahead and merged https://github.com/gatsbyjs/gatsby/pull/18551. So let's remove this change as we would end up with 2 exactly same lines.

There's still valuable change about `.env.development` file in this PR, so we still will merge that after this change

**Comment by divyanshu013** [comment]
Thanks for addressing @LekoArts. The related PR #18551 doesn't specify the node version to be LTS. During my local setup, I was unable to build on latest node 12, hence the original mention of LTS in this PR.

**Comment by LekoArts** [comment]
Good point! Feel free to add that note in the current sentence then 👍

**Pull request merged**